### PR TITLE
:apple: Update Mangeto X profile

### PR DESCRIPTION
**Pull request by mypeopoly**
- Add PET-CF and ABS-GF profile
- Update PETG-CF profile
- Add 0.6mm nozzle

### CODE REVIEW

File: resources/profiles/Peopoly/process/fdm_process_peopoly_common.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
     "tree_support_branch_angle": "45",
     "tree_support_wall_count": "0",
     "detect_thin_wall": "0",
-    "top_surface_pattern": "monotonicline",
+    "top_surface_pattern": "monotonic",
```
Comment (SoftFever) [inline resources/profiles/Peopoly/process/fdm_process_peopoly_common.json:87]: Orca now recommends using the monotonic line pattern for the top surface. Hence, I have updated all profiles to use the monotonic line recently.

I'm not sure if this change was intended or just an unintended result of being overwritten. I just wanted to mention it here

**Comment by SoftFever** [comment]
Can you check my comments and also this error? Older user profiles based on 'Peopoly Generic PLA' for a 0.8mm nozzle won't load properly anymore with the proposed new changes.
Error: 5-23 10:21:50.971249] [0x00007f1512e79740] [error]   can not find parent for config /home/runner/work/OrcaSlicer/OrcaSlicer/resources/profiles/user/default/filament/Peopoly_Peopoly Generic PLA 0.8 nozzle_orca_test.json!

**Comment by SoftFever** [comment]
Hi,

Can I check what's the status of this PR?

**Comment by mypeopoly** [comment]
I will close this PR first and then try again

**Pull request closed without merging**